### ISSUE TITLE
Could you share the coco trained model for me?

### ISSUE THREAD

**Issue by lzhaozi** [chuanqi305/MobileNet-SSD#40]
Hi,
Thanks for your great work. The speed is much faster than the original SSD model. 
I am finetuning the model to Pedestrain detection but havenot get good results from the voc 73000 iterations model. Could you share the coco pretrained model, if it is convenient? I think maybe the coco pretrained model has more generality. And finetuning on that will get better result than finetuning on voc model.
Thanks again for you great work.

**Comment by lzhaozi** [comment]
I have trained a pretty good coco model yet. The loss drop to ~4.0 and the detection eval mAP of val2017 coco is above 30%. I use the newest train2017 coco for training, which is larger than train2014. I'm satisfied with the performance now. Thank you for your great work. Very thanks!!!

**Comment by dlyldxwl** [comment]
can you share with me some train details? my eval map is only 21%. thx

**Comment by hao522** [comment]
@lzhaozi  could you kindly share your trained models on coco? thx
